.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "03:00"
+  open-pull-requests-limit: 10
```

Comment:
Not sure how we make this unlimited - or do we just increase to 100?

Reply:
I think increase to a higher number.

Reply:
Can we schedule dependabot to run on Sundays only?

Reply:
Good spot, the default would have been Monday @ 0500, but I have changed to Sunday